canonical-websites/www.ubuntu.com
# Created /about/packages page

## Done

- created new /about/packaging page
- the /about section is very simple, I didn't think it needed design or UX really; however, unless you wanted to add a debian, snap and charm icon?  up to you really @frankiegranato or @anasereijo

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/about/packages](http://0.0.0.0:8001/about/packages)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- compare to the [copy doc](https://docs.google.com/document/d/12PIhHMZZNzwLYCN1sLb1DSX2BejIy6b9-lhupghzNbY/edit?ts=5b7ec54a#heading=h.vyfp6mz8vnc9)


## Issue / Card

Fixes #3963
